app/views/admin/site_customization/content_blocks/_form.html.erb

```diff
@@ -17,7 +17,7 @@
   <div class="row">
     <div class="small-12 medium-6 column">
       <%= f.label :name %>
-      <%= f.select :name, SiteCustomization::ContentBlock::VALID_BLOCKS, label: false %>
+      <%= f.select :name, SiteCustomization::ContentBlock::VALID_BLOCKS.map { |key| [t("admin.site_customization.content_blocks.content_block.names.#{key}"), key] }, label: false %>
```

Comment:
@decabeza while i named the keys as suggested, the orignial reason to why they were named as such was a language issue. I did some thing like this to pull the wording from the `admin.yml` file.

Will that be OK or shall I just include it in our fork?

Reply:
It's OK @ziyan-junaideen 😉 

Run `i18n-tasks health`, maybe `admin.site_customization.content_blocks.content_block.names.` appears as unused one. Sometimes there is to add dynamic keys to `config/i18n-tasks.yml` under `ignore_missing` or `ignore_unused` 😌

Reply:
I am not sure what you are saying. Could you elaborate. The `admin.site_customization.content_blocks.content_block.names` is used to give humanized text to users.

How ever I only updated the `en` file. One reason is I am not sure if it will need changes. The other is English is the only language I am familiar with and will have to use Google to translate the text.

    Forest (en) has 2951 keys in total. On average, values are 25 characters long, keys have 3.9 segments.
    ✓ Perfect! No translations are missing.
    ✓ Well done! Every translation is in use.
    The following data requires normalization:
    config/locales/en/general.yml
    config/locales/en/activerecord.yml
    config/locales/en/activemodel.yml
    config/locales/en/admin.yml
    config/locales/en/moderation.yml
    config/locales/en/valuation.yml
    config/locales/en/management.yml
    config/locales/en/verification.yml
    config/locales/en/mailers.yml
    config/locales/en/pages.yml
    config/locales/en/devise_views.yml
    config/locales/en/responders.yml
    config/locales/en/kaminari.yml
    config/locales/en/officing.yml
    config/locales/en/budgets.yml
    config/locales/en/legislation.yml
    config/locales/en/community.yml
    config/locales/en/documents.yml
    config/locales/en/images.yml
    config/locales/en/guides.yml
    Run `i18n-tasks normalize` to fix

Reply:
Sometimes if you use an i18n key generated dynamically `content_blocks.content_block.names.#{key}` i18n health thinks there unused keys, but it seems is not the case.

Could you please include this keys on `es/admin.yml`? (we always include this two locales, don't worry for use Google translate) 😉 

```yml
names:
  top_links: Enlaces superiores
  footer: Pie de página
  subnavigation_left: Navegación principal izquierda 
  subnavigation_right: Navegación principal derecha
```

Thank you very much again! 🙏

Reply:
I thought it was the case when i checked the libs docs. I saw a flag to enable dynamic keys. It will support up to one wild card key.

I will add the keys.